lmdeploy/turbomind/turbomind.py

```diff
@@ -93,30 +92,6 @@ def _update_tm_config(dst: TurbomindModelConfig, src: TurbomindEngineConfig):
     return TurbomindModelConfig.from_dict(dst_dict)
 
 
-def _compare_individual_gpu_memory(tp: int):
```

Comment:
Hi @zhyncs 
we are planning to remove this code snippet for the following reasons:
- `torch.cuda.set_device(i)` affects vlm inference with tp enabled.
- turbomind(csrc) already handled the memory discrepancy